rfcs/text/0004/session.yml

```diff
@@ -0,0 +1,85 @@
+---
+- name: session
+  title: Session
+  group: 2
+  type: group
+  short: User, admin, application, network, or service sessions
+  description: |-
+    These fields are used to track an entity's interaction with various assets, services, and applications in an enterprise.  Sessions will typically include a start event, often a login / authorization event performed locally or via network based mechanisms, and an end event indicating a logoff or session termination.  Related events during the scope of the session will typically be associated via tuples of user, source and or destination ip/port, or cookies.
+
+    When available, event start/end or duration fields should be populated, as well as iam, user, network, host, observer, process, source, destination, client, and server fields as appropriate to describe the specifics of the interaction.
+
+  fields:
+
+    - name: kind
+      level: extended
+      type: keyword
+      short: Session kind (system, application or network).
+      description: >
+        Session kind describes the type of access represented. Acceptable values are:
+        "application", "system", and "network".
+
+        "application" is relevant for application-level sessions such as web, ftp, database.
+
+        "system" is relevant for sessions initiated by computer systems rather than humans.
+
+        "network" is relevant to describe various kinds of network-level sessions
+        (VPN, 802.1x, wireless WPA sessions, NAC access to network).
+
+      example: network
+
+    - name: type
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      short: 'Session type (array of: local, remote, virtual, wired, wireless and/or vpn).'
+      description: >
+        Session type provides additional details on the scope of the interactions being tracked. Acceptable values are: "local", "remote", "virtual", "wired", "wireless", "vpn".
+
+        "local" is relevant e.g. for serial connections, physical sessions (hands on keyboard).
+
+        "remote" is relevant for any remote connection: direct system access via remote desktop, ssh, etc.
+
+        "virtual" is for VDI, VDA, etc.
+
+        (missing clarifications for: wired, wireless and vpn network connections?)
```

Comment:
Forgot to bring attention to this description.

When I reformatted it, I noticed we were missing definitions for wired, wireless and vpn. Could you add those please?